core/classes/Misc/IntegrityChecker.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+// str_starts_with() is PHP 8, polyfill for PHP 7.4 compatibility
+// source: Laravel Framework
+// https://github.com/laravel/framework/blob/8.x/src/Illuminate/Support/Str.php
+if (!function_exists('str_starts_with')) {
```

Comment:
This should already be polyfilled via https://github.com/NamelessMC/Nameless/blob/develop/composer.json#L26

Reply:
Thanks